AUTHORS.md

```diff
@@ -160,4 +160,5 @@ sravya91
 srcopela
 tobadavid
 vfrancesmolla
+Zcaic
```

Comment:
Because of how Git outputs this list, names with upper case come first, move to after "Zan Xu" please.